sljit_src/sljitProtExecAllocator.c

```diff
@@ -98,19 +98,8 @@ struct chunk_header {
 #define O_TMPFILE 020200000
 #endif
 
-#ifdef __NetBSD__
-/*
- * this is a workaround for NetBSD < 8 that lacks a system provided
- * secure_getenv function.
- * ideally this should never be used, as the standard allocator is
- * a preferred option for those systems and should be used instead.
- */
-#define secure_getenv(name) issetugid() ?  NULL : getenv(name)
-#else
-char *secure_getenv(const char *name);
-#endif
-
 #ifndef _GNU_SOURCE
+char *secure_getenv(const char *name);
```

Comment:
I would keep this line out of this `#ifndef _GNU_SOURCE` ifdef in order to prevent unintentional changing of the code for !NetBSD.

Reply:
it is just a function definition for whoever forgets to build this on linux with EXTRA_CPPFLAGS='-D_GNU_SOURCE'

it doesn't change the code anywhere, except avoiding a warning and some silly bugs for whoever ignores those when it tries to convert the string into an int

Reply:
I see, then this is OK. It's also not a NetBSD code block, so I'm relaxed with it myself.